### ISSUE TITLE
Please take down this solution!

### ISSUE THREAD

**Issue by atzeus** [pelledrijver/IJVM-emulator#1]
You have various solutions to course assignments on your github page.

This is interfering with our courses: students are looking at and maybe copying part of your code.

Notice also that you are not legally allowed to post this code, as the copyright of the frameworks is not yours. Could you please take them down or make them private as soon as possible? Thanks!

Regards,

Atze

**Comment by pelledrijver** [comment]
Dear Atze,

Sorry for the inconvenience. Before I take this repository down, I first want to explain myself to hopefully change your mind. First of all, in the README I specify that the framework is part of the university course and that I've only written the c and header files in the src folder. I don't claim anywhere that the framework is written by myself. Secondly, I've been encouraged multiple times by professors to manage my code with GitHub. During the course, I've always kept the repository private until the course ended and all grades were released. I've spoken to multiple TA's about this topic and they told me I was allowed to keep my projects on GitHub as long as I made them private again during the same periods in the next years when new students have to code similar assignments.

When students crawl the web to find answers to their assignments and copy people's code from GitHub, I think that that's mostly their mistake. Nevertheless, I still make several repositories private each period because I don't want to be involved in fraud accusations.

I understand if I still have to take these projects down due to copyright claims, but I really hope it doesn't have to come to this. It hasn't been pointed out in any course.

With kind regards,
Pelle
